modules/management-console-agent/src/main/java/org/apache/ignite/agent/Agent.java

```diff
@@ -274,6 +274,12 @@ private void connect() {
             return;
         }
 
+        if (F.isEmpty(cfg.getConsoleUris())) {
+            log.info("Skip start Management Console agent on coordinator, because the server URI is not set");
```

Comment:
We should show message that will help user. Current message will confuse.

Reply:
Done